docs/pages/deploy-a-cluster/deployments/aws-starter-cluster-terraform.mdx

```diff
@@ -0,0 +1,714 @@
+---
+title: Teleport Single Instance deployment on AWS
+description: How to quickly configure Teleport on a single instance for testing in AWS.
+tocDepth: 3
+---
+
+This guide is designed to accompany our [reference starter-cluster Terraform code](https://github.com/gravitational/teleport/tree/branch/v(=teleport.major_version=)/examples/aws/terraform/starter-cluster#teleport-terraform-aws-ami-simple-example)
```

Comment:
I would briefly describe the infrastructure components that this Terraform module deploys so a user can determine whether the module and this guide are appropriate for their use case.